### DISCUSSION TITLE
Save as a new draft option for non-publishing users

### DISCUSSION THREAD

**Discussion by manibae** [craftcms/cms#13055] [Ideas] [answered]
It would be great for the users who don't have the "Save entries" (publish) permission to be able to create a new draft from an already published entry. 

Very often I get asked if users without publish permission could create a new draft from an enabled (published) entry for the better productivity. 

I think this was suggested previously [here](https://github.com/craftcms/cms/discussions/9634)

Thanks.

**Comment by brandonkelly** [comment]
This is already possible ✨

If a user has permission to View an entry, then they will see a “Create a draft” button at the top, where they can work on edits to the entry.

![An Edit Entry page, for an entry the user doens’t have permission to save, with an arrow pointing to the “Create a draft” button.](https://user-images.githubusercontent.com/47792/230218631-63b0912a-6690-4545-a166-a15cbcd340cc.png)

Actually publishing the changes in that draft will require the help of another user with higher permissions, of course.

**Comment by manibae** [comment]
Hi Brandon, 

Thanks for the info. 

It was not about creating a draft for the published entry. 
It was for creating an another new draft (like save as new) from a published entry. So the new draft would have a different URL. 

Hope this makes sense. 

Thanks. 

||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
Mani Bae

William Joseph
20–22 Wenlock Road
London N1 7GU

T 020 7637 5077

williamjoseph.co.uk

**Reply by brandonkelly** [reply]
Ah ok, sorry I misunderstood. That’s actually already in the works for Craft 5 (PR’d at #12303). I’ll update this once that’s done.

**Comment by brandonkelly** [comment, marked as answer]
We just got to this for Craft 5 (https://github.com/craftcms/cms/pull/12303) 🎉 Going forward, any user with the “Create entry” permission will have the option to “Save as a new entry” for any entry they have permission to view, and the newly-created entry will start off in an unpublished draft state rather than a fully-saved entry.

**Comment by manibae** [comment]
Happy new year!
That sounds great. Thanks for the update. 😃
